src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenIterators.vb

```diff
@@ -1830,5 +1830,28 @@ BC35000: Requested operation is not available because the runtime library functi
                 </expected>)
         End Sub
 
+        <Fact>
+        <WorkItem(220696, "https://devdiv.visualstudio.com/defaultcollection/DevDiv/_workitems#_a=edit&id=220696")>
+        Public Sub IteratorPropertySetterDebugInfo()
```

Comment:
I guess the name is about trying to emit with debug options. Consider merging the two tests and adding an attempt to compile with release options as well. Consider also running and observing behavior. Both for setter and getter.